src/m365/spo/commands/user/user-remove.spec.ts

```diff
@@ -303,7 +427,174 @@ describe(commands.USER_REMOVE, () => {
     assert(correctRequestIssued);
   });
 
-  it('handles error when removing using from web', async () => {
+  it('removes email by username successfully without prompting with confirmation argument', async () => {
+    let removeRequestIssued = false;
+
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if ((opts.url as string).indexOf(`_api/web/siteusers/GetByEmail('${formatting.encodeQueryParameter(validEmail)}')`) > -1) {
```

Comment:
At some point we decided not to use `indexOf` on part of url when doing stub of an request. We had one issue where we only checked part of request and tests were fine but as a result they would fail if we would check the all url. 
So let's try to make those tests a bit more bulletproof and use `===` instead of `indexOf()` and compare to the full url of the request.

Reply:
this comment also applies to the new tests added below 🙂

Reply:
I have replaced the indexof in all the test cases.